src/Adapters/AzureManagementAdapter.ts

```diff
@@ -20,7 +21,7 @@ import {
 } from '../Models/Constants';
 import { createGUID, getDebugLogger } from '../Models/Services/Utils';
 
-const MAX_RESOURCE_TAKE_LIMIT = 1000;
+// const MAX_RESOURCE_TAKE_LIMIT = 1000; // if necessary limit the number of resources to check against permissions
```

Comment:
do we want to remove this if it's not used?

Reply:
Since now we are using storageContainerAccountId parameter to pass to the container picker it is not crashing and we don't need to slice the resources result. But I wanted to keep it as comments for the future if any of the other resource pickers crashes, users can try to slice the resources before making another batch of calls for role assignments.